### ISSUE TITLE
Card: This H4 creates an inappropriate jump in heading levels within the document heading hierarchy

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3819]
Description: This H4 creates an inappropriate jump in heading levels within the document heading hierarchy

Code Snippet:
<h4 _ngcontent-exn-c558="" fd-card-second-subtitle="" class="fd-card__second-subtitle">Q1, 2019</h4>

Expected: Ensure that all heading elements are used in the proper order. For example, a heading level two should generally not be preceded by a heading level four just as a heading level 2 should not be preceded by a heading level 4.
Headings are not required to have sub-heading, for example, a heading level 2 section need not have any heading below it. A rule of thumb is that in general the heading level should match the visual representation of the heading level.
There are some exceptions to this rule. For example, when a navigation section(s) or comes before the main content in the reading order an h2 element may be used to identify the navigation area. This h2 would come before the h1 which is the heading for the main content of the page.
It's possible to have a h2 followed by an h4 if the h4 represents content that is perhaps lower in structure or importance than a h3 where an h3 might be used elsewhere in the same article. However, if the h4 is really the next logical level after h2 and doesn't match an h4 visually from another part of the same article or is not of lesser importance such as used in an error message or aside then it should be an h3 to communicate the structure.
Due to lack of support for the HTML5 sectional structure pages should continue to use the corresponding HTML numbered heading levels (h1,h2, h3, etc.). ARIA role of "heading" and aria-level can also be used to properly order the numbered heading levels in HTML5 outline structure.

screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98773178-648c7200-240e-11eb-9e2e-7a7c3dedf966.png)

**Comment by DeepakSap14** [comment]
issue is not clear. 
As explanation says.. h2 should come before h4.
This is what is happening currently. so i think this is correct behaviour.

Please update clearly if i am missing something.

**Comment by DeepakSap14** [comment]
confirmed with accessibility team. This issue is not present anymore. current behaviour is correct.
so this can be closed.
